assignment4/countingIslands_test.py

```diff
@@ -0,0 +1,42 @@
+"""Tests for countingIslands."""
+
+from disjointset import DisjointSet
+from countingIslands import countingIslands
+import unittest
+
+class countingIslandsTest(unittest.TestCase):
+    def testBaseCases(self):
+        islandsMap = [[False, True,  False, True],
```

Comment:
Optional:
To make this easier to read, you could write a function that converts a string representation of the map to the proper format.

For example:
```python
>>> mapOf(".L.L|"
...       "LL..|"
...       "..L.|"
...       "..L.")
[[False, True,  False, True], [True,  True,  False, False], [False, False, True,  False], [False, False, True,  False]]
```